currency/internal/gen.go

```diff
@@ -0,0 +1,3 @@
+package internal
+
+//go:generate go run cmd/main.go && gofmt -w ../std.go
```

Comment:
I see now, related to my comment above, that you are running the generation file using go generate.